crates/astria-cli/src/commands/bridge/collect.rs

```diff
@@ -199,10 +199,21 @@ async fn block_to_actions(
         .await
         .wrap_err_with(|| {
             format!(
-                "failed getting actions for block; block hash: `{block_hash}`, block height: \
+                "failed getting logs for block; block hash: `{block_hash}`, block height: \
                  `{rollup_height}`"
             )
-        })?;
+        })?
+        .into_iter()
+        .filter_map(|r| {
```

Comment:
I am ambivalent about changing this logic in the CLI at this point. I am fine with adding a flag that filters out bad actions, but I would leave this to a followup.